mobile_app_inventory/static/www/app.js

```diff
@@ -78,7 +78,7 @@ angular.module(
 
     $ionicConfigProvider.views.transition('none');
 
-    $urlRouterProvider.otherwise('/login');
+    $urlRouterProvider.otherwise('/inventory/');
```

Comment:
Sorry but I reverted this change you introduced during the port of this module.
This was creating error if user was not correctly logged. Did you fixed the problem ?